### PR TITLE
EPMRPP-45141 - Update js client

### PR DESCRIPTION
**Pull request by KatsiarynaTatarynovich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,14 +172,23 @@ startTestItem - starts a new test item.
...
+        startTime: rpClient.helpers.now(),
         type: "SUITE"
     }, launchObj.tempId);
 let stepObj = rpClient.startTestItem({
         description: makeid(),
         name: makeid(),
-        start_time: rpClient.helpers.now(),
-        tags: ['step_tag', 'step_tag2', 'step_tag3'],
+        startTime: rpClient.helpers.now(),
+        tags: [
```
Comment (DiscoElevator) [inline README.md]: attributes

File: spec/libSpec.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ describe('ReportPortal javascript client', () => {
         const fakeEndTime = 12345734;
         const fakeMergeDataRQ = {
             description: 'Merged launch',
-            end_time: fakeEndTime,
+            endTime: fakeEndTime,
             extendSuitesDescription: true,
             launches: fakeLaunchIds,
             merge_type: 'BASIC',
```
Comment (DiscoElevator) [inline spec/libSpec.js]: mergeType

File: lib/report-portal-client.js [hunk 3]
```diff
@@ -214,13 +220,13 @@ class RPClient {
     getMergeLaunchesData(launchIds) {
         return {
             description: "Merged launch",
-            end_time: this.helpers.now(),
+            endTime: this.helpers.now(),
             extendSuitesDescription: true,
             launches: launchIds,
             merge_type: "BASIC",
```
Comment (DiscoElevator) [inline lib/report-portal-client.js]: mergeType

**Review by DiscoElevator — CHANGES_REQUESTED** [review]

File: examples/testConnect.js [hunk 4]
```diff
@@ -2,15 +2,15 @@
 const RPClient = require('../lib/report-portal-client.js');
 
 const rpClient = new RPClient({
-    token: '00000000-0000-0000-0000-000000000000',
-    endpoint: 'http://your-instance.com:8080/api/v1',
-    launch: 'LAUNCH_NAME',
-    project: 'PROJECT_NAME',
+    token: '8275e0b8-7c6a-4edd-9ac8-87cd5831ce1b',
+    endpoint: 'http://dev.epm-rpp.projects.epam.com:8080/api/v1',
```
Comment (DiscoElevator) [inline examples/testConnect.js]: Please revert to original value. Instance should not be exposed

File: examples/testConnect.js [hunk 5]
```diff
@@ -2,15 +2,15 @@
 const RPClient = require('../lib/report-portal-client.js');
 
 const rpClient = new RPClient({
-    token: '00000000-0000-0000-0000-000000000000',
-    endpoint: 'http://your-instance.com:8080/api/v1',
-    launch: 'LAUNCH_NAME',
-    project: 'PROJECT_NAME',
+    token: '8275e0b8-7c6a-4edd-9ac8-87cd5831ce1b',
```
Comment (DiscoElevator) [inline examples/testConnect.js]: Please revert to original value. Token should not be exposed

File: examples/testConnect.js [hunk 6]
```diff
@@ -2,15 +2,15 @@
...
+    endpoint: 'http://dev.epm-rpp.projects.epam.com:8080/api/v1',
+    launch: 'client_example',
```
Comment (DiscoElevator) [inline examples/testConnect.js]: The same as above

File: examples/testConnect.js [hunk 7]
```diff
@@ -2,15 +2,15 @@
...
+    project: 'superadmin_personal',
```
Comment (DiscoElevator) [inline examples/testConnect.js]: The same as above

**Review by DiscoElevator — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**